internal/helpers/default_domain.go

```diff
@@ -1,10 +1,19 @@
 package helpers
```

Comment:
I think we should not build on the pattern to have generic packages like "helpers", "common", ... but try to find a good scope for new code and also move old code out of this packages.